api/client/gro_client.py

```diff
@@ -634,23 +634,12 @@ def main():
         sys.exit(0)
     client = GroClient(API_HOST, access_token)
 
-    selected_entities = {}
-    if args.item:
-        selected_entities['item_id'] = client.search_for_entity('items', args.item)
-    if args.metric:
-        selected_entities['metric_id'] = client.search_for_entity('metrics', args.metric)
-    if args.region:
-        selected_entities['region_id'] = client.search_for_entity('regions', args.region)
-    if args.partner_region:
-        selected_entities['partner_region_id'] = client.search_for_entity('regions',
-                                                                          args.partner_region)
-    if not selected_entities:
-        selected_entities = client.pick_random_entities()
```

Comment:
By not picking random entities, we still get a "random" result if no arguments are given, but it's just the first result that get_data_series returns which is consistent over time. So if I run this without arguments a bunch of times back to back I'll just keep getting

```py
{'metric_id': 15741072, 'metric_name': 'Density', 'item_id': 9125, 'item_name': 'Bulk density (fine earth) at 60 cm deep', 'region_id': 10776, 'region_name': "P'yongan-bukto", 'partner_region_id': 0, 'partner_region_name': 'World', 'frequency_id': 15, 'source_id': 89}
```

A bit less fun than the actually picking a random item each time

Reply:
fun restored